cirq-google/cirq_google/engine/virtual_engine_factory.py

```diff
@@ -84,6 +94,35 @@ def _create_perfect_calibration(device: cirq.Device) -> calibration.Calibration:
     return calibration.Calibration(calibration=snapshot, metrics=all_metrics)
 
 
+def load_median_device_calibration(processor_id: str) -> calibration.Calibration:
+    """Loads a median `cirq_google.Calibration` for the given device.
+
+    Real calibration data from Google's 'rainbow' and 'weber' devices has been
+    saved in Cirq. The calibrations selected are roughly representative of the
+    median performance for that chip.
+
+    A description of the stored metrics can be found here:
+    https://quantumai.google/cirq/google/calibration
```

Comment:
Can you doublecheck to make sure this is correct formatting?

See Dave's recent PR https://github.com/quantumlib/Cirq/pull/5220
Not sure if this needs an `{:.external}` if it is on quantumai.

Reply:
Switched to match formatting. It seems like skipping the `{:.external}` is in the spirit of the TFdocs rule, though I don't know if the warnings Dave mentioned will see it that way :/